static/src/javascripts/projects/common/modules/audio/inArticlePlayer.js

```diff
@@ -20,16 +20,35 @@ const showStarterBlockOnFirstPlay = () => {
     const scrubberBar: ?HTMLElement = document.querySelector(
         '.inline-audio_content_progress-bar'
     );
+    const loadingSpan: ?HTMLElement = document.querySelector(
+        '#loading-message'
+    );
     if (scrubberBar) scrubberBar.classList.add('started');
+    if (loadingSpan) loadingSpan.classList.add('loading');
+};
+
+const removeLoadingState = el => {
+    el.classList.remove('loading');
 };
 
 // PLAYER ACTION FUNCTIONS
 
-const updateTime = (el: HTMLElement, player, progressBar: HTMLElement) => {
+const updateTime = (
+    el: HTMLElement,
+    player,
+    progressBar: HTMLElement,
+    loadingSpan: HTMLElement
+) => {
     let prevTime = 0;
+
     player.addEventListener('timeupdate', () => {
         el.textContent = formatTime(Math.round(player.currentTime));
         const percentPlayed = (player.currentTime / player.duration) * 100;
+
+        // how can i only do this once?
+        if (loadingSpan.classList.contains('loading')) {
+            removeLoadingState(loadingSpan);
+        }
```

Comment:
@regiskuckaertz - can you think of a more optimal way to do this? I don't want check a Dom element classList on every DOM update - only the very first time. Is there a `once` function?